.github/workflows/examples-k32w.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-k32w:0.5.99
+            image: connectedhomeip/chip-build-k32w:0.6.02
```

Comment:
If this is required, yaml in integrations/cloudbuild/ have to be updated as well (the build-all and smoke test)